use_cases/eluc/app/components/prediction.py

```diff
@@ -22,27 +23,28 @@ class PredictionComponent:
     def __init__(self, df):
         self.df = df
         self.predictors = self.load_predictors()
-        # TODO: This needs to change to a percent change predictor when next PR is merged
-        self.prescriptor_manager = PrescriptorManager(None, None)
+        self.change_predictor = PercentChangePredictor()
 
     def load_predictors(self) -> dict[str, Predictor]:
         """
         Loads in predictors from disk, downloads from HuggingFace first if needed.
         TODO: Currently hard-coded to load specific predictors. We need to make this able to handle any amount!
         :return: dict of predictor name -> predictor object.
         """
+        nn_persistor = HuggingFacePersistor(NeuralNetSerializer())
+        sklearn_persistor = HuggingFacePersistor(SKLearnSerializer())
         predictors = {}
         nn_path = "danyoung/eluc-global-nn"
         nn_local_dir = app_constants.PREDICTOR_PATH / nn_path.replace("/", "--")
         linreg_path = "danyoung/eluc-global-linreg"
         linreg_local_dir = app_constants.PREDICTOR_PATH / linreg_path.replace("/", "--")
         rf_path = "danyoung/eluc-global-rf"
         rf_local_dir = app_constants.PREDICTOR_PATH / rf_path.replace("/", "--")
-        global_nn = NeuralNetPredictor.from_pretrained(nn_path,
+        global_nn = nn_persistor.from_pretrained(nn_path,
                                                     local_dir=nn_local_dir)
-        global_linreg = LinearRegressionPredictor.from_pretrained(linreg_path,
+        global_linreg = sklearn_persistor.from_pretrained(linreg_path,
                                                                 local_dir=linreg_local_dir)
-        global_rf = RandomForestPredictor.from_pretrained(rf_path,
+        global_rf = sklearn_persistor.from_pretrained(rf_path,
```

Comment:
Models are loaded with `Persistor.from_pretrained` now instead of `model.from_pretrained`